src/app/src/util/util.js

```diff
@@ -195,7 +195,9 @@ export const createParamsFromQueryString = (qs) => {
     } = querystring.parse(qsToParse);
 
     if (status) {
-        return Object.freeze({ status });
+        return Array.isArray(status)
+            ? Object.freeze({ status })
+            : Object.freeze({ status: [status] });
```

Comment:
Consider adding updating the test to check for this new outcome.

Reply:
Added tests in fixup b1e998a